tests/lib/rules/max-len.js

```diff
@@ -309,6 +314,17 @@ ruleTester.run("max-len", rule, {
                 }
             ]
         }, {
+            code: "// A comment that exceeds the max comment length, but not the specified max length for code.",
+            options: [40, 4, { comments: 80 }],
```

Comment:
I think the issue is that the text of the comment in the test case is misleading; it says that it exceeds the max comment length, but it actually exceeds the max code length.